README.md

```diff
@@ -1,30 +1,46 @@
+<div style="text-align:center;">
```

Comment:
```suggestion
<div align="center">
```
This is how to do alignment according to https://gist.github.com/nikhilnayyar002/7a35e653d3d590e317c829243e73b110#allignments